spec/features/borrower_manages_loan_request.rb

```diff
@@ -1,11 +1,17 @@
 require "rails_helper"
 
-RSpec.feature "borrower creates loan request" do
+RSpec.feature "borrower manages loan request" do
   let(:borrower) { User.create(name: "Richard",
                                email: "richard@example.com",
                                password: "password",
                                role: "borrower")
-                   }
+  }
+
+  let(:lender) { User.create(name: "Rachel",
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.